head/src/main/kotlin/io/qalipsis/core/head/campaign/PersistentCampaignService.kt

```diff
@@ -65,7 +66,8 @@ internal class PersistentCampaignService(
     private val campaignConfigurationConverter: CampaignConfigurationConverter,
     private val campaignConverter: CampaignConverter,
     private val factoryRepository: FactoryRepository,
-    private val campaignScheduler: CampaignScheduler
+    private val campaignScheduler: CampaignScheduler,
+    private val hooks: List<CampaignHook>
```

Comment:
Do not forget to also add the hooks to the other implementation of CampaignService.